server/src/main/java/fr/uga/l3miage/tp2/exo1/models/ClubSportifEntity.java

```diff
@@ -0,0 +1,19 @@
+package fr.uga.l3miage.tp2.exo1.models;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name = "ClubSportif")
```

Comment:
pas de `camelCase` dans les nom en base
```suggestion
@Table(name = "club_sportif")
```